apps/web/src/constants/routes.enum.ts

```diff
@@ -9,6 +9,7 @@ export enum ROUTES {
   PARTNER_INTEGRATIONS_VERCEL_LINK_PROJECTS = '/partner-integrations/vercel/link-projects',
   PARTNER_INTEGRATIONS_VERCEL_LINK_PROJECTS_EDIT = '/partner-integrations/vercel/link-projects/edit',
   ANY = '/*',
+  TEMPLATES_DIGEST_PLAYGROUND = '/templates/digest-playground/:templateId',
```

Comment:
I think we are carried away because of previous implementations but URL should be: `'/templates/:templateId/digest-playground'` to be more REST compliant.

Reply:
sure I can do that :)